packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -866,17 +884,19 @@ export class AzureCommunicationCallWithChatAdapter implements CallWithChatAdapte
           adapter.off('error', listener);
         });
         break;
+      case 'chatInitialized':
+        this.emitter.off(event, listener);
+        break;
       default:
         throw `Unknown AzureCommunicationCallWithChatAdapter Event: ${event}`;
     }
   }
 
-  private async executeWithResolvedChatAdapter(callback: (adapter: ChatAdapter) => void): Promise<void> {
-    if (this.chatAdapter) {
-      callback(this.chatAdapter);
+  private executeWithResolvedChatAdapter(callback: (adapter: ChatAdapter) => void): void {
```

Comment:
Make sure to revert function signatures like `dispose` back to `void` instead of `Promise<void>`